hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -596,7 +617,14 @@
       clock_group: "infra",
       reset_connections: {rst_ni: "sys"},
       base_addrs: {rom: "0x00008000", regs: "0x411e0000"}
-    }
+    },
+    { name: "rv_core_ibex_peri",
```

Comment:
I suppose instantiating this in the top rather than inside `rv_core_ibex` is easier from tooling / automation point of view, at the expense of increased integration DV impact?

Reply:
Ooh, I take that back - the DV impact is the same because `rv_core_ibex` is not verified. Ok with instantiating this at the top if that's easier.

Reply:
yeah mostly because there's a lot of automation stuff missing. We don't currently the hosts, we would have to add that in so that we auto-wire tlul, alerts, interrupts etc. 

So this is just a cop-out solution to fit into our existing infrastructure.